assets/src/dashboard/app/views/storyAnimTool/timeline/components.js

```diff
@@ -36,7 +36,7 @@ export const AnimationList = styled.div(
     width: 100%;
     height: 100%;
     overflow: scroll;
-    background-color: ${theme.DEPRECATED_THEME.colors.gray25};
```

Comment:
These changes are to non-user-facing UIs, so I just chose the most similar colors from the current theme.